src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Resources/Base/BuildReleaseArm64SimpleDotNet.apkdesc

```diff
@@ -98,5 +98,5 @@
       "Size": 1904
     }
   },
-  "PackageSize": 2693653
+  "PackageSize": 2714133
```

Comment:
It looks like a Xamarin.Forms (legacy) app is about 77kb bigger and hello world android 20kb bigger.

That seems reasonable if there is concern of something breaking from this setting.